packages/cactus-plugin-ccmodel-hephaestus/src/main/typescript/pm4py-adapter/check_conformance.py

```diff
@@ -0,0 +1,226 @@
+import sys
```

Comment:
@RafaelAPB I recommend storing Python source code under `./src/main/python` instead of `./src/main/typescript/`

Reply:
@brunoffmateus please move the python code to this other folder please

Reply:
@RafaelAPB I forgot to mention that this was done as well.